.chronus/changes/InfoUnallowKeyChecker-2024-8-24-11-6-11.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@typespec/openapi"
+---
+
+@info decorator failed to exclude extension key not starting with "x-".
```

Comment:
```suggestion
`@info` decorator validate no extra properties not starting with `x-` are provided.
```
escape decorators in the changelog or it will tag the user with that name in the release